packages/panw_cortex_xdr/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.2.0"
+  changes:
+    - description: Add event datastream to collect event forwarding logs via the Google Cloud Storage input and add dashbaords for alert and incident data-stream.
```

```suggestion
    - description: Add event data stream to collect event forwarding logs via the Google Cloud Storage input and add dashboards for alert and incident data-streams.
```
(sorry, I missed these)